src/main/NWN/API/Object/NwObject.cs

```diff
@@ -172,9 +172,9 @@ public LocalUUID GetLocalUUID(string name)
       return new LocalUUID(this, name);
     }
 
-    public bool HasClashingUUID()
+    public bool HasUUID()
     {
-      return string.IsNullOrEmpty(NWScript.GetObjectUUID(this));
+      return string.IsNullOrEmpty(NWNX.API.Object.PeekUUID(this));
     }
```

Comment:
I'll have to test this - I think this function was to check if there was a GUID conflict, not that it did not have one.

Reply:
From my understanding, if a conflict happens it will trigger the nwnxee event for a uuid collision. 

So if a copy of an item is sent to a player, and the uuids match, as soon as the second uuid comes into being it triggers that event, and gives the new/copied item a new uuid.